installation/cmd/run.sh

```diff
@@ -20,3 +20,7 @@ echo "Secrets with Tiller tls client certificates have been created \n"
 
 MINIKUBE_IP=$(eval minikube ip)
 helm install --set=global.minikubeIP=${MINIKUBE_IP} --set=global.isLocalEnv=true --name "${COMPASS_HELM_RELEASE_NAME}" --namespace "${COMPASS_HELM_RELEASE_NAMESPACE}" "${ROOT_PATH}"/chart/compass --tls --wait
+
+# TODO: Remove it after next CLI release
+echo "Adding Compass entries to /etc/hosts...\n"
```

Comment:
Could you please remove `\n`? I've just noticed this mistake, the `\n` is printed in the console. I know it's not your fault, but we would be grateful for the change 😄 Thanks!